### PR TITLE
[DRAFT] Two Arm : Teleop and Policy

### PR DESCRIPTION
**Pull request by Pythonista7**
## What this does
This PR contains changes from our (me and @1nf0rmed ) lerobot hack which enables two arm (+ 3cams [left-wrist,right-wrist , global top view] ) teleop and recording of training data. We also trained a policy which controls the setup. Will be sharing huggingface datasets and models soon! 

Dataset : https://huggingface.co/datasets/1nF0rmed/two-arm-coordination

Pretrained Policies Coming soon!! (We are working on OpenVLA and ACT at the moment, planning more experiments soon.)

Our [X post/video here](https://x.com/Ash7Win10/status/1934229493763441119)

Examples:
|  Title               | Label           |
|----------------------|-----------------|
| Fixes #[issue]       | (🐛 Bug)        |
| Adds new dataset     | (🗃️ Dataset)    |
| Optimizes something  | (⚡️ Performance) |

## How it was tested
Explain/show how you tested your changes.

Examples:
- Added `test_something` in `tests/test_stuff.py`.
- Added `new_feature` and checked that training converges with policy X on dataset/environment Y.
- Optimized `some_function`, it now runs X times faster than previously.

## How to checkout & try? (for the reviewer)
Provide a simple way for the reviewer to try out your changes.

Examples:
```bash
pytest -sx tests/test_stuff.py::test_something
```
```bash
python lerobot/scripts/train.py --some.option=true
```

## SECTION TO REMOVE BEFORE SUBMITTING YOUR PR
**Note**: Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR. Try to avoid tagging more than 3 people.

**Note**: Before submitting this PR, please read the [contributor guideline](https://github.com/huggingface/lerobot/blob/main/CONTRIBUTING.md#submitting-a-pull-request-pr).

### CODE REVIEW

**Comment by imstevenpmwork** [comment]
Hi @Pythonista7, thanks for sharing your hackathon work! Two-arm SO-10x teleop and recording is now supported natively: `bi_so_follower` / `bi_so_leader` landed in #2780 and work with the standard `lerobot-teleoperate` and `lerobot-record` CLIs, including per-arm wrist cameras plus a shared top-view camera. The paths this draft modifies (`lerobot/common/...`) were also removed in the repo restructure, and the draft deletes test artifacts, so it can no longer be rebased meaningfully. Closing as superseded — please give the native bimanual setup a try and open an issue if anything from your workflow is missing!
